frontend/src/app/products/product-mgmt-dashboard/product-mgmt-dashboard.component.css

```diff
@@ -0,0 +1,31 @@
+
+form>button {
+    margin-left: 30px;
+  }
+  
+  #left {
+    float: left;
```

Comment:
Today's solution for this sort of layouts is best done with flexbox. [Guide Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)
There's a fun way to learn flexbox with a zombie game: https://mastery.games/flexboxzombies/